source/includes/_sub_category_search_metrics.md

```diff
@@ -0,0 +1,158 @@
+            "cp_click_16": 0.8285518,
+            "cp_click_83": 2.0893919,
+            "cp_lead_avg": 3.2245355,
+            "cp_lead_std_dev": 1.1849484,
+            "cp_lead_median": 3.1994891,
+            "cp_lead_16": 1.899118,
+            "cl_lead_83": 4.5574494,
+            "cp_call_avg": 3.5352362,
+            "cp_call_std_dev": 1.1795996,
+            "cp_call_median": 3.6362695,
+            "cp_call_16": 2.0955609,
+            "cp_call_83": 4.8079478,
+            "ctr_avg": 0.0971623,
+            "ctr_std_dev": 0.0635744,
+            "ctr_median": 0.08084,
+            "ctr_16": 0.05068,
+            "ctr_83": 0.13217,
+            "ctl_avg": 5.9390747,
+            "ctl_std_dev": 3.3800717,
+            "ctl_median": 5.07692,
+            "ctl_16": 2.6,
+            "ctl_83": 9.99999,
+            "cp_qual_lead_avg": 3.4885108,
+            "cp_qual_lead_std_dev": 1.1449597,
+            "cp_qual_lead_median": 3.6101068,
+            "cp_qual_lead_16": 2.0943302,
+            "cp_qual_lead_83": 4.6171976,
+            "budget_avg": 1756.7639257,
+            "budget_std_dev": 907.119281,
+            "budget_median": 1500.0,
+            "budget_16": 800.0,
+            "budget_83": 3000.0,
+            "last_modified": "2023-05-04 19:01:08"
+          },
+          {
+            "business_sub_category_id": 114,
+            "business_sub_category_name": "Photography",
+            "cp_click_avg": 1.3923322,
+            "cp_click_std_dev": 0.6058227,
+            "cp_click_median": 1.2725656,
+            "cp_click_16": 0.8285518,
+            "cp_click_83": 2.0893919,
+            "cp_lead_avg": 3.2245355,
+            "cp_lead_std_dev": 1.1849484,
+            "cp_lead_median": 3.1994891,
+            "cp_lead_16": 1.899118,
+            "cl_lead_83": 4.5574494,
+            "cp_call_avg": 3.5352362,
+            "cp_call_std_dev": 1.1795996,
+            "cp_call_median": 3.6362695,
+            "cp_call_16": 2.0955609,
+            "cp_call_83": 4.8079478,
+            "ctr_avg": 0.0971623,
+            "ctr_std_dev": 0.0635744,
+            "ctr_median": 0.08084,
+            "ctr_16": 0.05068,
+            "ctr_83": 0.13217,
+            "ctl_avg": 5.9390747,
+            "ctl_std_dev": 3.3800717,
+            "ctl_median": 5.07692,
+            "ctl_16": 2.6,
+            "ctl_83": 9.99999,
+            "cp_qual_lead_avg": 3.4885108,
+            "cp_qual_lead_std_dev": 1.1449597,
+            "cp_qual_lead_median": 3.6101068,
+            "cp_qual_lead_16": 2.0943302,
+            "cp_qual_lead_83": 4.6171976,
+            "budget_avg": 1756.7639257,
+            "budget_std_dev": 907.119281,
+            "budget_median": 1500.0,
+            "budget_16": 800.0,
+            "budget_83": 3000.0,
+            "last_modified": "2023-05-04 19:01:08"
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+if there is no campaign found, the below empty campaign array will be returned:
```

Comment:
Our other campaigns behave differently, so unless there is a reason you need this behavior I suggest you choose behavior similiar to

```
curl --location --globoff 'https://data-connect-staging.gannettdigital.com/client_reports/facebook_campaign/TEST_1?start_date=2023-10-28&end_date=2023-11-26&global_master_campaign_id[]=TEST_12127' \
--header 'Authorization: TRUSTED_TOKEN'
```

returns a 403 with the following body
```
[
    {
        "name": "report parameters",
        "message": "one or more invalid campaigns specified (does not exist, or not valid for this advertiser)"
    }
]
```

that should be a 422 but that isn't relevant.  Do what everyone else does and you don't have to write code.

Reply:
An empty array represents what we are looking for so it's correct